recipe/meta.yaml

```
@@ -1,35 +1,50 @@
-{% set version = "0.6.2" %}
+{% set name = "parse_type" %}
+{% set version = "0.6.6" %}
 
 package:
-  name: parse_type
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/parse_type/parse_type-{{ version }}.tar.gz
-  sha256: 79b1f2497060d0928bc46016793f1fca1057c4aacdf15ef876aa48d75a73a355
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 513a3784104839770d690e04339a8b4d33439fcd5dd99f2e4580f9fc1097bfb2
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: true  # [py<27]
 
 requirements:
   host:
     - python
     - setuptools
```

https://github.com/jenisys/parse_type/blob/v0.6.6/pyproject.toml#L10
```suggestion
    - setuptools >=64
```